File: project1/src/main/project1Scala/pipeline1/Task1.scala
package pipeline1

import org.apache.spark.sql.SparkSession

object Task1 {
  def run(spark: SparkSession, inputPath: String, outputPath: String): Unit = {
    val inputData = spark.table(inputPath)

    inputData
      .write
      .mode("overwrite")
      .saveAsTable(outputPath)
  }
}

File: project1/src/main/project1Python/pipeline1/task1.py
from pyspark.sql import SparkSession

def run_task(spark: SparkSession, input_path: str, output_path: str):
    print(f"Reading from {input_path}")
    input_data = spark.read.table(input_path)

    print(f"Writing to {output_path}")
    input_data.write.mode("overwrite").saveAsTable(output_path)

    print("Task completed successfully.")